sycl/include/CL/sycl/detail/pi.h

```diff
@@ -1819,9 +1823,9 @@ struct _pi_plugin {
   // Some choices are:
   // - Use of integers to keep major and minor version.
   // - Keeping char* Versions.
-  char PiVersion[4];
+  char PiVersion[5];
```

Comment:
why don't we increase it to a bigger value now, say 10?

Reply:
Good call. I have increased it to 10. That should give us plenty room for future version increases.